spec/requests/users_spec.rb

```diff
@@ -1,10 +1,33 @@
 require 'rails_helper'
 
 RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    it 'returns http success' do
-      get '/users/index'
-      expect(response).to have_http_status(:success)
-    end
+  it 'returns http success' do
+    get '/users/'
+    expect(response).to have_http_status(:success)
+  end
+
+  it 'to have template index for all users' do
+    get '/users/'
+    expect(response).to render_template(:index)
+  end
+
+  it 'Body incudes correct placeholder text: Users' do
+    get '/users/'
+    expect(response.body).to include('Users')
+  end
```

Comment:
- [ ] [Optional] It would be better professionally if you include these 3 test cases for users `index` inside `describe` method like that:
```ruby
describe 'GET users#index' do
  it 'returns http success' do
    get '/users/'
    expect(response).to have_http_status(:success)
  end

  it 'to have template index for all users' do
    get '/users/'
    expect(response).to render_template(:index)
  end

  it 'Body incudes correct placeholder text: Users' do
    get '/users/'
    expect(response.body).to include('Users')
  end
end
```

Reply:
Thanks